christfellowshipchurch/web-app-v2
# Set a Reminder for Spanish

### About
This PR checks the location and if it is one of the Spanish locations it will display the copy for the modals in Spanish.

### Test Instructions
1. Test Setting a Remainder in a Spanish location and a non Spanish location.
2. Ensure the copy in both are as expected, including buttons.

### Screenshots
|Spanish|English|
|-|-|
|<img width="947" alt="Screenshot 2024-08-08 at 2 59 13 PM" src="https://github.com/user-attachments/assets/cb5be5df-0805-4241-a625-9427a137b574">|<img width="952" alt="Screenshot 2024-08-08 at 2 58 49 PM" src="https://github.com/user-attachments/assets/7b284607-372b-4d19-b4d0-3f30c0ccfb4a">|

### Closes Tickets
[CFDP-2986](https://christfellowshipchurch.atlassian.net/browse/CFDP-2986)


[CFDP-2986]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2986?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
